docs/live-query.rst

```diff
@@ -0,0 +1,100 @@
+Live Query
+==========
+
+With Live Query, you can ask questions of endpoints and quickly identify areas for improving security and IT hygiene.
+
+You can use recommended queries created by Carbon Black security experts or craft your own SQL queries. Live Query is
+powered by https://osquery.io, an open source project that uses an SQLite interface. This guide will get you started
+using Live Query via the Python SDK.
+
+More information about the Audit and Remediation product which uses Live Query is available in the
+`Carbon Black Cloud user guide
+<https://docs.vmware.com/en/VMware-Carbon-Black-Cloud/services/carbon-black-cloud-user-guide/GUID-129D4F84-1BF0-49F3-BF95-83002FD63217.html/>`_
+
+More information about Live Query APIs is available on the `Developer Network <https://developer.carbonblack.com/reference/carbon-black-cloud/cb-liveops/>`_.
+
+Overview
+--------
+This guide shows how to find specific files on a system. This is the same scenario as the Quick Start Guide for the
+APIs on the `Developer Network <https://developer.carbonblack.com/reference/carbon-black-cloud/cb-liveops/latest/livequery-api/>`_
+
+The steps we'll go through are:
+
+#. Set up the python imports and Carbon Black Cloud credentials
+#. Start the Query Run
+#. Get the Query Run by ID to check the status
+#. Get the results of the query
+
+Setting up
+----------
+
+The code snippets assume that the python environment has been set up with the necessary imports and credentials.
+
+    >>> from cbc_sdk import CBCloudAPI
+    >>> from cbc_sdk import audit_remediation
+    >>> from cbc_sdk.audit_remediation import Run, RunHistory, Result, ResultQuery
+    >>> api = CBCloudAPI(profile='sample')
+
+For more information on credential handling in the SDK, see :doc:`authentication`
+
+Start a Query Run
+-----------------
+Set up the query for the file you are looking for.  Then create a query object, execute it, and get the id of the run.
+
+    >>> query_string = "SELECT filename, path FROM file WHERE path = 'C:\\Windows\\Temp\\dbutil_2_3.sys\\' OR path LIKE 'C:\\Users\\%\\AppData\\Local\\Temp\\dbutil_2_3.sys';"
+    >>> query_object = api.select(Run).where(sql=query_string)
+    >>> run = query_object.submit()
+    >>> print(run.id)
+    vsc2be500dcuhc1q5bhvq7kdwoqh367i
+
+Get the Query Run to check status
```

Comment:
For my context what is the intention  behind this section as you already have the run from the previous section as that is returned from the `submit()`. Are we needing  to refresh the object to  get the update status? We could use `run.refresh()`

Reply:
Changed to run.refresh()